index.js

```diff
@@ -57,11 +69,15 @@ class TemplateCompiler extends Filter {
     utils.initializeEmberENV(templateCompiler, EmberENV);
   }
 
+  precompile(content, options) {
+    return this.options.templateCompiler.precompile(content, options);
+  }
+
   processString(string, relativePath) {
-    return 'export default ' + utils.template(this.options.templateCompiler, stripBom(string), {
-      contents: string,
-      moduleName: relativePath
-    }) + ';';
+    return 'export default Ember.HTMLBars.template(' + this.precompile(stripBom(string), {
```

Comment:
Can you break this up a bit so that we don't have so many invocations? Just thinking that we could make a few variables, and then use simple template string to concat...

(I know this isn't a new problem with this PR though...)